src/ucc/api/ucc.h

```diff
@@ -659,6 +661,25 @@ typedef struct ucc_oob_coll {
 
 typedef ucc_oob_coll_t ucc_context_oob_coll_t;
 typedef ucc_oob_coll_t ucc_team_oob_coll_t;
+
+/**
+ *
+ *  @ingroup UCC_CONTEXT_DT
+ */
+typedef struct ucc_mem_map {
+    void *   address; /*!< the address of a buffer to be attached to the NIC */
```

Comment:
@manjugv @wfaderhold21  why word "NIC" here? Attached to ucc context maybe ?
Minor comment: you call it a "map" but the data struct looks exactly like "struct iovec". Would ucc_iov_t be a better name ?